<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>220</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer/>
    </item>
    <item>
     <widget class="QLineEdit" name="username_edit">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_edit">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="text">
       <string>ورود</string>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>